tjmonopix2/system/bdaq53.py

```diff
@@ -82,6 +82,12 @@ def init(self, **kwargs):
         #     for _ in range(100):
         #         self.rx_channels['rx0'].get_rx_ready()
 
+    def enable_ext_trigger(self):
+        self['cmd'].set_ext_trigger(True)
+
+    def disable_ext_trigger(self):
+        self['cmd'].set_ext_trigger(False)
+
```

Comment:
Why is this needed? As far as I see, this is only relevant for RD53A chips with AZ flavors, according to the firmware. Anyway, there are no trigger commands sent to the chip.

Reply:
I don't know to be honest. But i will put this to the list and when the setup is working reliably, we can test removing it and see if there is any effect (probably not)